packages/esm-patient-common-lib/src/siderail-nav-button/siderail-nav-button.scss

```diff
@@ -38,13 +38,13 @@
   .countTag {
     position: absolute;
     font-size: 10px;
-    text-align: center;
+    text-align: right;
     color: $ui-01;
     background-color: $danger;
     padding: 0 4px;
     border-radius: 50%;
-    top: -10px;
-    right: -10px;
+    top: -14px;
+    right: -12px;
```

Comment:
Why is this included in the diff?

Reply:
I'm not sure why, because I believe this change was merged in sometime back

Reply:
I've squashed the commits and removed this from the diff.